python/triqs/lattice/utils.py

```diff
@@ -246,3 +246,161 @@ def TB_from_pythTB(ptb):
                     orbital_names=[str(i) for i in range(ptb.get_num_orbitals())])
 
     return TBL
+
+# importing necessary dependencies
+from itertools import product as itp
+from pythtb import *
+# from triqs.lattice.tight_binding import TBLattice # TRIQS needs to be installed for this dependency to work
+import sympy as sp
+from sympy import *
```

Comment:
It would be good to include import module statements needed for the function in the function itself in the beginning, and more general things like itertools, and warnings at the top of the `uitls.py` file.

Reply:
Fixed.